presto-hive/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -1728,15 +1734,9 @@ public T getJsonSerializableData()
         }
 
         @JsonProperty
-        public String getUser()
+        public HdfsContext getContext()
```

Comment:
`HdfsContext` is not JSON serializable. However, I don't think `Action` needs to be. Additionally, the `getJsonSerializableData()` method is never used. @haozhun was this something added during development that turned out not to be needed?